### PR TITLE
Responsive and cross browser fixes

### PR DESCRIPTION
**Pull request by hammerfunk**
IE is still under development though

### CODE REVIEW

File: app/assets/stylesheets/admin/base.scss [hunk 1]
```diff
@@ -167,6 +212,20 @@
     position: relative;
     overflow: hidden;
 
+    .ie & {
+      padding: 0;
+      border: none;
+      background: transparent !important;
+      overflow: visible;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered padding, border, background, box-shadow, overflow

File: app/assets/stylesheets/admin/base.scss [hunk 2]
```diff
@@ -1,6 +1,37 @@
...
 
 // Basic
+a {
+  &:active,
+  &:focus {
+    outline: none !important;
+  }
+}
+
+span.visible-lg {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid qualifying class selectors with an element.

File: app/assets/stylesheets/admin/base.scss [hunk 3]
```diff
@@ -1,6 +1,37 @@
...
+  @include screen-lg-min {
+    display: inline !important;
+  }
+}
+
+span.visible-md {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid qualifying class selectors with an element.

File: app/assets/stylesheets/admin/base.scss [hunk 4]
```diff
@@ -1,6 +1,37 @@
...
+  @include screen-md-min {
+    display: inline !important;
+  }
+}
+
+span.visible-sm {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid qualifying class selectors with an element.

File: app/assets/stylesheets/admin/base.scss [hunk 5]
```diff
@@ -1,6 +1,37 @@
...
+  @include screen-sm-min {
+    display: inline !important;
+  }
+}
+
+span.visible-xs {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Avoid qualifying class selectors with an element.

**Comment by RuslanHamidullin** [comment]
@hammerfunk looks like here is the small error:
```
Invalid CSS after "}": expected "}", was ""
         (in /home/travis/build/bitzesty/qae/app/assets/stylesheets/application-admin.scss:1071)
```

**Pull request merged**